test/util/factories/mock_trip_factory.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:motis_mitfahr_app/util/search/position.dart';
+import 'package:motis_mitfahr_app/util/trip/trip.dart';
+
+import 'trip_factory.dart';
+
+class MockTrip extends Trip {
```

Comment:
I'm not feeling great about two things: 

- this file/factory does not fit with the others
- the fact that we need to repeat all fields here _again_... 

Maybe the best solution is just to remove the `abstract` keyword from Trip, and then use the `TripFactory` for our purposes. I think that would solve a lot of complication, and `abstract` is currently really unnecessary since `Trip` implements all its methods.

What do you think @Eddie-42 ?

Reply:
I have pushed my idea on another branch. Please see this commit: https://github.com/MOTIS-Mitfahr-App/flutter-app/commit/283025be1f69fc9670e73eb2b3ae855b10b15adc

I think it's a lot better?

Reply:
I agree